src/joystick/hidapi/SDL_hidapi_gamecube.c

```diff
@@ -195,6 +203,8 @@ HIDAPI_DriverGameCube_InitDevice(SDL_HIDAPI_Device *device)
         }
     }
 
+    useRumbleBrake = SDL_GetHintBoolean(SDL_HINT_GAMECUBE_RUMBLE_BRAKE);
```

Comment:
I think the callback is called inside AddHintCallback so that variables like this get initialized ASAP, worth a check

Reply:
I'll take a look, I also missed the fact that I apparently deleted SDL_FALSE anyway